Tests/AsyncHTTPClientTests/HTTPClientTestUtils.swift

```diff
@@ -1548,3 +1586,88 @@ private let key = """
     oYQsPj00S3/GA9WDapwe81Wl2A==
     -----END PRIVATE KEY-----
     """
+
+final class BasicInboundTrafficShapingHandler: ChannelDuplexHandler {
```

Comment:
This turned out to not be necessary, but it's a test-only utility which can't hurt I thought :).